hack/generate/samples/internal/go/memcached-with-customization/memcached_with_customization.go

```diff
@@ -1273,40 +1298,6 @@ generate-metricsdocs:
 	go run -ldflags="${LDFLAGS}" ./monitoring/metricsdocs > docs/monitoring/metrics.md
 `
 
-const webhooksFragment = `
-// TODO(user): change verbs to "verbs=create;update;delete" if you want to enable deletion validation.
-//+kubebuilder:webhook:path=/validate-cache-example-com-v1alpha1-memcached,mutating=false,failurePolicy=fail,sideEffects=None,groups=cache.example.com,resources=memcacheds,verbs=create;update,versions=v1alpha1,name=vmemcached.kb.io,admissionReviewVersions=v1
-
-var _ webhook.Validator = &Memcached{}
-
-// ValidateCreate implements webhook.Validator so a webhook will be registered for the type
-func (r *Memcached) ValidateCreate() (admission.Warnings, error) {
-	memcachedlog.Info("validate create", "name", r.Name)
-
-	return nil, validateOdd(r.Spec.Size)
-}
-
-// ValidateUpdate implements webhook.Validator so a webhook will be registered for the type
-func (r *Memcached) ValidateUpdate(old runtime.Object) (admission.Warnings, error) {
-	memcachedlog.Info("validate update", "name", r.Name)
-
-	return nil, validateOdd(r.Spec.Size)
-}
-
-// ValidateDelete implements webhook.Validator so a webhook will be registered for the type
-func (r *Memcached) ValidateDelete() (admission.Warnings, error) {
-	memcachedlog.Info("validate delete", "name", r.Name)
-
-	return nil, nil
-}
-func validateOdd(n int32) error {
-	if n%2 == 0 {
-		return errors.New("Cluster size must be an odd number")
-	}
-	return nil
-}
```

Comment:
@acornett21 if we want the validating webhook here then we need to add the option to scaffold this type and then just replace TODO:(user) implement your logic for the logic that we want instead.